nav2_amcl/src/amcl_node.hpp

```diff
@@ -38,6 +38,9 @@
 #include "nav2_util/sensors/laser/laser.hpp"
 #include "nav2_util/motion_model/motion_model.hpp"
 #include "nav2_util/angleutils.hpp"
+#include "rclcpp/parameter_events_filter.hpp"
+#include "nav2_dynamic_params/dynamic_params_validator.hpp"
```

Comment:
Are you sing the validator class anywhere?

Reply:
Not at the moment.  Removed.